ui/lib/api-common-protos/google/api/annotations_pb.js

```diff
@@ -2,7 +2,6 @@
 /**
  * @fileoverview
  * @enhanceable
- * @suppress {missingRequire} reports error on implicit type usages.
```

Comment:
[boulder]

I’m ready to approve this, but I’m a bit confused by the removal of these `@suppress` lines.

Can we compare tool versions? Here’s mine:

| Tool | Version | Package Manager |
| --- | --- | --- |
| `protoc` | `3.17.3` | Homebrew |
| `ts-protoc-gen` | `0.15.0` | Volta |
| `protoc-gen-grpc-web` | `1.2.1` | Homebrew |

Reply:
|`protoc` | `3.14.0`| Binary (manual install) |
| `ts-protoc-gen` | `0.15.0`| yarn |
|`protoc-gen-grpc-web ` |`1.2.1`| Binary (manual install) |

FWIW I've started seeing the removals in https://github.com/hashicorp/waypoint/pull/1739 , but was not expecting my setup to do the same. I wonder what has changed.

Reply:
Interestingly, when I run `make gen/ts` with protoc `3.17.3` I don’t see any change to those `@suppress {missingRequire}` lines. So I suspect the `@suppress {missingRequire}` stuff was introduced to the core protoc javascript/closure-compiler generator some time *after* `3.14.0`. Could you try upgrading to `3.17.3` and see if the behavior changes?

Reply:
@jgwhite just did so, the `missingRequire` lines are missing no more.